Chutzpah/TestRunner.cs

```diff
@@ -684,8 +684,9 @@ private static string BuildRunnerArgs(TestOptions options, TestContext context,
             string runnerArgs;
             var testModeStr = testExecutionMode.ToString().ToLowerInvariant();
             var timeout = context.TestFileSettings.TestFileTimeout ?? options.TestFileTimeoutMilliseconds ?? Constants.DefaultTestFileTimeout;
-
-            runnerArgs = string.Format("--ignore-ssl-errors=true --proxy-type=none --ssl-protocol=any \"{0}\" {1} {2} {3} {4} {5}",
+            var proxySetting = string.IsNullOrEmpty(options.Proxy) ? (string.IsNullOrEmpty(context.TestFileSettings.Proxy)?"--proxy-type=none": string.Format("--proxy={0}", context.TestFileSettings.Proxy)) : string.Format("--proxy={0}",options.Proxy);

```

Comment:
@sunnyy999 This would be cleaner if you first declared a variable var proxy = options.Proxy ?? context.TestFileSettings.Proxy; and then used that in the proxySetting line. That would fix the extra level of ?; operators.